wallet/src/actors/app/error.rs

```diff
@@ -115,12 +115,18 @@ impl From<actors::worker::Error> for Error {
             actors::worker::Error::KeyGen(e @ crypto::Error::InvalidKeyPath(_)) => {
                 validation_error(field_error("seedData", e.to_string()))
             }
-            actors::worker::Error::Repository(_e @ repository::Error::InsufficientBalance) => {
-                validation_error(field_error(
-                    "balance",
-                    "Wallet account has not enough balance",
-                ))
-            }
+            actors::worker::Error::Repository(repository::Error::InsufficientBalance {
+                total_balance,
+                available_balance,
+                transaction_value,
+            }) => validation_error(field_error(
+                json! {{
+                    "total_balance": total_balance,
+                    "available_balance": available_balance,
+                    "transaction_value": transaction_value,
+                }},
+                "Wallet account has not enough balance",
+            )),
```

Comment:
This was the easiest way to send an object to the frontend, but this json object is serialized as a string, so the result is really ugly, JSON inside JSON:

    {"code":400,"message":"Validation Error","data":[["{\"available_balance\":17221973255,\"total_balance\":17221973255,\"transaction_value\":18000000853}","Wallet account has not enough balance"]]}

I don't know how to add additional fields to this error message, so any better alternatives welcome.